### PR TITLE
Output local files for k8s config and ssh config

### PR DESCRIPTION
**Pull request by tei-k**
# Description

Output files for `ssh config`, `inventory`, `kube config`

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by feeblefakie** [comment]
@ymorimo Merging it but PTAL when you get a chance!

File: modules/azure/network/locals.tf [hunk 1]
```diff
@@ -58,3 +58,36 @@ locals {
...
+[bastion]
+%{for f in module.bastion.bastion_host_ips~}
+${f}
+%{endfor}
+
+[bastion:vars]
+host=bastion
+
+[all:vars]
+internal_domain=${local.network.internal_domain}
```
Comment (ymorimo) [inline modules/azure/network/locals.tf:87]: @tei-k I got an error here.

```
Error: Unsupported attribute

  on .terraform/modules/network/modules/azure/network/locals.tf line 87, in locals:
  87: internal_domain=${local.network.internal_domain}
    |----------------
    | local.network is object with 9 attributes

This object does not have an attribute named "internal_domain".
```

Maybe `var.internal_domain` is correct?

Comment (tei-k) [reply modules/azure/network/locals.tf:87]: Thanks! fixed in #289

**Pull request merged**